release-notes/opensearch-dashboards.release-notes-2.13.0.md

```diff
@@ -27,6 +28,7 @@
 - [Multiple Datasource] Export DataSourcePluginRequestContext at top level for plugins to use ([#6108](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6108))
 - [Multiple Datasource] Improves connection pooling support for AWSSigV4 clients in data sources ([#6135](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6135))
 - [Multiple Datasource] Add datasource version number to newly created data source object([#6178](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6178))
+- [Multiple Datasource] Add default functionality for customer to choose default datasource ([#6186](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6186))
```

Comment:
This is so ambiguous. Also, we have users and not customers.

Reply:
@zhyuanqi is working on it.

Reply:
Will make the change

Reply:
Updated the changes in this PR https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6228